aten/src/ATen/detail/AcceleratorHooksInterface.h

```diff
@@ -50,6 +50,20 @@ struct TORCH_API AcceleratorHooksInterface {
     return -1;
   }
 
+  virtual c10::Stream getCurrentStream(DeviceIndex device) const {
+    TORCH_CHECK(false, "Backend doesn't support getCurrentStream()");
+    return c10::Stream::unpack3(-1, 0, c10::DeviceType::CPU);
+  }
+
+  virtual c10::Stream getDefaultStream(DeviceIndex device) const {
+    TORCH_CHECK(false, "Backend doesn't support getDefaultStream()");
+    return c10::Stream::unpack3(-1, 0, c10::DeviceType::CPU);
+  }
+
+  virtual void setCurrentStream(const c10::Stream& stream) const {
```

Comment:
`setCurrentStream` may need to assign the `stream` to another stream variable. However, the `stream` is constant.